packages/compass-components/src/index.ts

```diff
@@ -83,6 +83,7 @@ export { VisuallyHidden } from '@react-aria/visually-hidden';
 export { openToast, closeToast, ToastArea } from './hooks/use-toast';
 
 export { breakpoints, spacing } from '@leafygreen-ui/tokens';
+export { H3, Body } from '@leafygreen-ui/typography';
```

Comment:
We're already exporting these from the `leafygreen.tsx` file:
https://github.com/mongodb-js/compass/blob/7371e1b9784fc9915bb9f38b9d13fcec236e190e/packages/compass-components/src/components/leafygreen.tsx#L162
So we can remove this.
```suggestion
```